tests/telemetry/test_series.py

```diff
@@ -0,0 +1,48 @@
+import time
+
+from ddtrace.internal.telemetry.data.host import get_hostname
+from ddtrace.internal.telemetry.data.metrics import Series
+
+
+def test_series_to_dict():
+    series = Series("test.metric")
+
+    assert series.to_dict() == {
+        "metric": "test.metric",
+        "points": [],
+        "tags": {},
+        "type": Series.COUNT,
+        "common": False,
+        "interval": None,
+        "host": get_hostname(),
+    }
+
+
+def test_series_add_tag():
+    series = Series("test.metric")
+
+    series.add_tag("foo", "bar")
+    assert series.tags == {"foo": "bar"}
```

Comment:
what happens if you set a tag that already exists?

Reply:
I don't see us testing `to_dict` when there are tags set.

Reply:
I added a test case for this. We should overwrite the existing tag. Each tag should map to one value